.gitignore

```diff
@@ -9,12 +9,14 @@
 *.pvtu
 *.so
 *.vtu
+.DS_Store
```

Comment:
This doesn't seem like a relevant change?

Reply:
I guess I could spin it out: it's cos I was adding the firedrake.adjoint.rst ignore. (This is a file MacOS finder dumps everywhere).

Reply:
Not worth the effort, keep it as-is.